starport/services/network/request.go

```diff
@@ -0,0 +1,19 @@
+package network
+
+import (
+	"context"
+
+	launchtypes "github.com/tendermint/spn/x/launch/types"
+)
+
+// FetchRequests fetches the chain requests from SPN by launch id
+func (n Network) FetchRequests(ctx context.Context, launchID uint64) ([]launchtypes.Request, error) {
```

Comment:
```suggestion
func (n Network) Requests(ctx context.Context, launchID uint64) ([]launchtypes.Request, error) {
```

If makes sense can we avoid action names like Fetch and Get in method names?